integrated_channels/blackboard/transmitters/learner_data.py

```diff
@@ -42,7 +42,7 @@ def single_learner_assessment_grade_transmit(self, exporter, **kwargs):
             exporter: The learner completion data payload to send to blackboard
         """
         kwargs['app_label'] = 'blackboard'
-        kwargs['model_name'] = 'BlackboardLearnerDataTransmissionAudit'
```

Comment:
since we are fixing a bug I think it would be wise to capture this with a test, or do you think an existing test cover this? perhaps sth as simple as checking the right model is used in the test?